openapi.yaml

```diff
@@ -1186,6 +1187,14 @@ components:
         type: integer
         minimum: 0
       description: Offset of where search results start
+    offline:
+      name: offline
+      in: query
+      required: false
```

Comment:
For the future, `required: false` is redundant, as it is false by default. But otherwise LGTM.